src/plugin/ipc/ssh/ssh.cpp

```diff
@@ -337,6 +339,23 @@ static void prepareForExec(char *const argv[], char ***newArgv)
     prefix += " --ssh-slave ";
   }
 
+  if(dmtcp_pathvirt_enabled && dmtcp_pathvirt_enabled()) {
+    if(getenv(ENV_NEW_DPP)) {
```

Comment:
I'm a little uneasy with this solution based on passing information between plugins using environment vars. Couldn't we just use the `get_new_path_prefix_list()` API here?